.github/workflows/mattermost-ziti-webhook.yml

```diff
@@ -10,14 +10,25 @@ on:
   push:
   fork:
   release:
+    types: [released]
 
 jobs:
-  mattermost-ziti-webhook:
-    runs-on: macos-latest
-    name: POST Webhook
+  ziti-webhook:
+    runs-on: ubuntu-latest
+    name: Ziti Mattermost Action - Py
     steps:
-    - uses: openziti/ziti-webhook-action@main
+    - uses: openziti/ziti-mattermost-action-py@main
+      if: ${{ env.ZHOOK_URL != null }}
+      env:
+          ZHOOK_URL: ${{ secrets.ZHOOK_URL }}
       with:
-        ziti-id: ${{ secrets.ZITI_MATTERMOST_IDENTITY }}
-        webhook-url: ${{ secrets.ZITI_MATTERMOST_WEBHOOK_URL }}
-        webhook-secret: ${{ secrets.ZITI_MATTERMOSTI_WEBHOOK_SECRET }}
+        # Identity JSON containing key to access a Ziti network
+        zitiId: ${{ secrets.ZITI_MATTERMOST_IDENTITY }}
+
+        # URL to post the payload. Note that the `zitiId` must provide access to a service 
+        # intercepting `my-mattermost-ziti-server`
+        webhookUrl: ${{ secrets.ZHOOK_URL }}
+
+        eventJson: ${{ toJson(github.event) }}
+        senderUsername: "openziti/ngx_http_ziti_module"
```

Comment:
odd username choice.  It will be ignored for the moment by the action, and replaced with the sender indicated in the GitHub payload

Reply:
What is a valid username choice or should this input be ommitted?

Reply:
You can leave it.  Other ymls used "GitHubZ"